compiler/src/codegen/comp_utils.re

```diff
@@ -195,152 +165,147 @@ let type_of_repr = repr => {
 };
 
 let write_universal_exports =
-    (wasm_mod, {Cmi_format.cmi_sign}, exported_names) => {
-  Types.(
-    Type_utils.(
-      List.iter(
-        item => {
-          switch (item) {
-          | TSigValue(
-              id,
-              {
-                val_repr: ReprFunction(args, rets, direct),
-                val_fullpath: path,
-              },
-            ) =>
-            let name = Ident.name(id);
-            let exported_name = "GRAIN$EXPORT$" ++ name;
```

Comment:
Why is the `GRAIN$EXPORT$` prefix gone?

Reply:
We don't write special Grain exports anymore; now we only write the universal exports (see my other comment).